controllers/grid/settings/library/form/EditLibraryFileForm.php

```diff
@@ -26,16 +26,15 @@
 
 class EditLibraryFileForm extends LibraryFileForm
 {
-    /** @var LibraryFile the file being edited, or null for new */
-    public $libraryFile;
+    /** The file being edited, or null for new */
+    public ?LibraryFile $libraryFile;
 
-    /** @var int the id of the context this library file is attached to */
-    public $contextId;
+    /** The id of the context this library file is attached to */
+    public int $contextId;
```

Comment:
Since this attribute is already present in the parent class through constructor promotion, it shouldn't be re-declared here. The declaration and comment can be removed from this class.

Reply:
good catch! I've updated it